torchgen/api/python.py

```diff
@@ -201,6 +201,26 @@
 # For examples, only pyi signatures include return types.
 
 
+def format_function_signature(
+    name: str, arguments: Iterable[str] = (), return_type: str | None = None
+) -> str:
+    if not isinstance(arguments, (list, tuple)):
+        arguments = tuple(arguments)
+    return_type = f" -> {return_type}" if return_type is not None else ""
+
+    sig = f"def {name}({', '.join(arguments)}){return_type}: ..."
+    if len(sig) <= 80 or len(arguments) == 0 or tuple(arguments) == ("self",):
+        return sig
+
+    return "\n".join(
+        (
+            f"def {name}(",
+            *(f"    {arg}," for arg in arguments),
+            f"){return_type}: ...",
+        )
+    )
```

Comment:
This PR provides a standard way to create a formatted signature. Previously, we did it in a handcrafted way.